bundler.d/assets.rb

```diff
@@ -1,7 +1,8 @@
 gem 'jquery-rails', '~> 3.1'
 gem 'jquery-turbolinks', '~> 2.1'
 gem 'jquery-ui-rails', '< 5.0.0'
-gem 'bootstrap-sass', '3.0.3.0'
+gem 'bootstrap-sass'
+gem 'patternfly-sass'
```

Comment:
We need to pin gem 'autoprefixer-rails', '~> 5.2' here, 6.0+ is only Ruby 2.0 compatible and we still support 1.9.3